src/main/java/com/google/sps/datastore/GetDatastore.java

```diff
@@ -0,0 +1,74 @@
+package com.google.sps.datastore;
+
+import java.util.ArrayList;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+
+//PCAP storage class
+public class GetDatastore {
+
+    public ArrayList<PCAPdata> dataTable = new ArrayList<PCAPdata>();
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+  public GetDatastore() 
```

Comment:
Indentation is bad. Are you using some sort of Editor?

Reply:
Was doing most of my coding on Google cloud shell, not the best editor, Switch to personal laptop with VS code and reformatted code